README.md

```diff
@@ -1,2 +1,151 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 METRICS-WEBAPP <a name="about-project"></a>
+
+METRICS-WEBAPP is a website that display a list of cryptocurrency coins and their current market info..
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>REACT</li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **use gitflow**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Installation of Node.js, React and Redux
+- A working browser
+- Internet connection
+- A code editor like Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:katfogy/react-capstone-web-metric-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd capstone-web-metric-app
+  npm install
+```
+
+### Usage
+
+To use this project on your prefered browser use
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+```test
+npx eslint "**/*.{js,jsx}"
+npx stylelint "**/*.{css,scss}"
+npm test
+```
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://crypto-web-capstone.onrender.com/)
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Foga Kater Amos**
+
+- GitHub: [@githubhandle](https://github.com/katfogy)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/foga-amos-2b5371103/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Api calls]**
+- [ ] **[Functional Components]**
+- [ ] **[Redux ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/katfogy/react-capstone-web-metric-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give a star ⭐️ to this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this amazing project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Currently, the MIT license link is broken, kindly fix it by using the correct file name. I think you should use `LICENSE` instead of `LICENSE.md`. 🤝️